docs/content/quickstart/build-real-time-data-lake-tutorial.md

```diff
@@ -0,0 +1,292 @@
+   ```sql
+    CREATE DATABASE db_1;
+    USE db_1;
+    CREATE TABLE user_1 (
+      id INTEGER NOT NULL PRIMARY KEY,
+      name VARCHAR(255) NOT NULL DEFAULT 'flink',
+      address VARCHAR(1024),
+      phone_number VARCHAR(512),
+      email VARCHAR(255)
+    );
+    INSERT INTO user_1 VALUES (110,"user_110","Shanghai","123567891234","user_110@foo.com");
+   
+    CREATE TABLE user_2 (
+      id INTEGER NOT NULL PRIMARY KEY,
+      name VARCHAR(255) NOT NULL DEFAULT 'flink',
+      address VARCHAR(1024),
+      phone_number VARCHAR(512),
+      email VARCHAR(255)
+    );
+   INSERT INTO user_2 VALUES (120,"user_120","Shanghai","123567891234","user_120@foo.com");
+   ```
+   ```sql
+   CREATE DATABASE db_2;
+   USE db_2;
+   CREATE TABLE user_1 (
+     id INTEGER NOT NULL PRIMARY KEY,
+     name VARCHAR(255) NOT NULL DEFAULT 'flink',
+     address VARCHAR(1024),
+     phone_number VARCHAR(512),
+     email VARCHAR(255)
+   );
+   INSERT INTO user_1 VALUES (110,"user_110","Shanghai","123567891234", NULL);
+
+   CREATE TABLE user_2 (
+     id INTEGER NOT NULL PRIMARY KEY,
+     name VARCHAR(255) NOT NULL DEFAULT 'flink',
+     address VARCHAR(1024),
+     phone_number VARCHAR(512),
+     email VARCHAR(255)
+   );
+   INSERT INTO user_2 VALUES (220,"user_220","Shanghai","123567891234","user_220@foo.com");
+   ```
+
+## Creating tables using Flink DDL in Flink SQL CLI
+First, use the following command to enter the Flink SQL CLI Container:
+```shell
+docker-compose exec sql-client ./sql-client
+```
+
+We should see the welcome screen of the CLI client:
+
+![Flink SQL Client](/_static/fig/real-time-data-lake-tutorial/flink-sql-client.png  "Flink SQL Client" )
+
+Then do the following steps in Flink SQL CLI:
+
+1. Enable checkpoints every 3 seconds
+   
+   Checkpoint is disabled by default, we need to enable it to commit Iceberg transactions.
+   Besides, the beginning of mysql-cdc binlog phase also requires waiting a complete checkpoint to avoid disorder of binlog records.
+   ```sql
+   -- Flink SQL                   
+   Flink SQL> SET execution.checkpointing.interval = 3s;
+   ```
+2. Create MySQL sharding source table 
+
+   Create a source table that captures the data from the logical sharding table `user`. Here, we use regex to match all the physical tables.
+   Besides, the table defines metadata column to identify which database/table the record comes from.
+   ```sql
+   -- Flink SQL
+   Flink SQL> CREATE TABLE user_source (
+       database_name STRING METADATA VIRTUAL,
+       table_name STRING METADATA VIRTUAL,
+       `id` DECIMAL(20, 0) NOT NULL,
+       name STRING,
+       address STRING,
+       phone_number STRING,
+       email STRING,
+       PRIMARY KEY (`id`) NOT ENFORCED
+     ) WITH (
+       'connector' = 'mysql-cdc',
+       'hostname' = 'mysql',
+       'port' = '3306',
+       'username' = 'root',
+       'password' = '123456',
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)